Frameworks/CoreFoundationAdditions/_NSCFTemporaryRootObject.mm

```diff
@@ -16,4 +16,8 @@
 #include "_NSCFTemporaryRootObject.h"
 
 @implementation _NSCFTemporaryRootObject
+- (void)doesNotRecognizeSelector:(SEL)selector {
+    // According to the reference platform documentation, this method must never return.
+    abort();
```

Comment:
code after RaiseFailFastException is unreachable, so you should simply remove the abort.

---
In reply to: [71600334](https://github.com/Microsoft/WinObjC/pull/671#discussion_r71600334) [](ancestors = 71600334,71582230,71576020,71575314,71574896,71574472)